### PR TITLE
Exploratory notebook

### PR DESCRIPTION
**Pull request by devotuoma**
I have created an exploratory notebook for a random Erdos-any graph using 100 nodes and 0.3 edge probability and 0.6 edge probability. I have also created a barabasi_albert_graph and given inferences about the plots in the markdown. I accept comments and any recommendations.

### CODE REVIEW

File: 2023-round-1/devotuoma/nx_tutorial_script.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# importing networkx library
+import networkx as nx
+# printing the version once more
+print(nx.__version__)
+G = nx.DiGraph() # calling on the diagraph function using G as the global variable
+# Graph is always a collection of nodes along with identified pairs of nodes i.e. edges, links, e.t.c
+NetworkX, nodes can be any hashable object such as text string, an image, an XML object, another Graph, a customized node object, etc.
```
Comment (paulitapb) [inline 2023-round-1/devotuoma/nx_tutorial_script.py:7]:
```suggestion
#NetworkX, nodes can be any hashable object such as text string, an image, an XML object, another Graph, a customized node object, etc.
```

File: 2023-round-1/devotuoma/nx_tutorial_script.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+nx.draw(G, with_labels=True, font_weight='bold')
+
+subax2 = plt.subplot(122)
+
+nx.draw_shell(G, nlist=[range(5, 10), range(5)], with_labels=True, font_weight='bold')
+import matplotlib.pyplot as plt
+
+# PLotting the graph using networkx.drwa 
+nx.draw_networkx(G,with_labels=True,node_color=colors)
```
Comment (paulitapb) [inline 2023-round-1/devotuoma/nx_tutorial_script.py:56]: The list colors have a different length than the number of nodes in G. The list was created using the nodes of the first digraph G that have fewer nodes than the Petersen graph.

File: 2023-round-1/devotuoma/nx_tutorial_script.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+G = nx.petersen_graph()
+
+subax1 = plt.subplot(121)
+
+nx.draw(G, with_labels=True, font_weight='bold')
+
+subax2 = plt.subplot(122)
+
+nx.draw_shell(G, nlist=[range(5, 10), range(5)], with_labels=True, font_weight='bold')
+import matplotlib.pyplot as plt
```
Comment (paulitapb) [inline 2023-round-1/devotuoma/nx_tutorial_script.py:53]: This should be before you use `plt.subplot()` in line 46. Preferably all imports at the beginning of the file.

File: 2023-round-1/devotuoma/nx_tutorial_script.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
+# importing networkx library
+import networkx as nx
+# printing the version once more
+print(nx.__version__)
+G = nx.DiGraph() # calling on the diagraph function using G as the global variable
+# Graph is always a collection of nodes along with identified pairs of nodes i.e. edges, links, e.t.c
+NetworkX, nodes can be any hashable object such as text string, an image, an XML object, another Graph, a customized node object, etc.
+#incorporating nodes with attributes group and color based on types
+G.add_node(1)
+G.add_node(1, group='int',color='red')
```
Comment (paulitapb) [inline 2023-round-1/devotuoma/nx_tutorial_script.py:10]: Here you add node 1 twice. The resulting graph will not have 2 nodes labeled "1" but with different attributes.

**Review by paulitapb — CHANGES_REQUESTED** [review]
Just check the comments on the tutorial script

**Comment by devotuoma** [comment]
Okay am correcting this

**Comment by MridulS** [comment]
I merged https://github.com/networkx/outreachy/pull/148 so closing this. It's a duplicate

**Pull request closed without merging**